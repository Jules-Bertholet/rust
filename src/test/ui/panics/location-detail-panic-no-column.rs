// run-fail
// check-run-results
// compile-flags: -Zlocation-detail=line,file
// exec-env:RUST_BACKTRACE=0
// ignore-emscripten has extra panic output

fn main() {
    panic!("column-redacted");
}
