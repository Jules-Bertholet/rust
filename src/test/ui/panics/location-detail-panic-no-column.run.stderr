thread 'main' panicked at 'column-redacted', $DIR/location-detail-panic-no-column.rs:8:0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
